api/app/utils/redapp.py

```diff
@@ -0,0 +1,316 @@
+""" Call Java REDapp code.
+"""
+from datetime import datetime, date
+import jnius_config
+# import jnius - importing jnius on this level causes an segmentation fault.
+from app import config
+
+jnius_config.set_classpath(config.get('CLASSPATH'))
+
+
+class FBPCalculations:  # pylint: disable=missing-class-docstring, too-many-instance-attributes
+    """ Duplicate of Java class ca.cwfgm.fbp.FBPCalculations
+
+    public double ros_t;
+    public double ros_eq;
+    public double fros;
+    public double bros;
+    public double rso;
+    public double hfi;
+    public double ffi;
+    public double bfi;
+    public double area;
+    public double perimeter;
+    public double distanceHead;
+    public double distanceFlank;
+    public double distanceBack;
+    public double lb;
+    public double csi;
+    public double cfb;
+    public double sfc;
+    public double tfc;
+    public double cfc;
+    public double isi;
+    public double fmc;
+    public double wsv;
+    public double raz;
+    public double conifMixedWood;
+    public double deadBalsam;
+    public double grassCuring;
+    public double grassFuelLoad;
+    public double crownBase;
+    public double elapsedTime;
+    public double ffmc;
+    public double windSpeed;
+    public double windDirection;
+    public double bui;
+    public double dmc;
+    public double dc;
+    public double slopeValue;
+    public double aspect;
+    public double latitude;
+    public double longitude;
+    public double elevation;
+    public boolean acceleration;
+    public boolean useCrownBaseHeight;
+    public boolean useBui;
+    public boolean useBuildup;
+    public boolean useSlope;
+    public boolean useGreenup;
+    public int fuelType;
+    public String fireDescription;
+    public String headFireDescription;
+    public String flankFireDescription;
+    public String backFireDescription;
+
+    public Calendar m_date;
+
+    public boolean cfbPossible = true;
+    """
+
+    def __init__(self):  # pylint: disable=too-many-statements
+        """ Init variables """
+        # pylint: disable=invalid-name
+        self.ros_t: float = None
+        self.ros_eq: float = None
+        self.fros: float = None
+        self.bros: float = None
+        self.rso: float = None
+        self.hfi: float = None
+        self.ffi: float = None
+        self.bfi: float = None
+        self.area: float = None
+        self.perimeter: float = None
+        self.distanceHead: float = None
+        self.distanceFlank: float = None
+        self.distanceBack: float = None
+        self.lb: float = None
+        self.csi: float = None
+        self.cfb: float = None
+        self.sfc: float = None
+        self.tfc: float = None
+        self.cfc: float = None
+        self.isi: float = None
+        self.fmc: float = None
+        self.wsv: float = None
+        self.raz: float = None
```

Comment:
do you happen to know what `wsv` and `raz` are?

Reply:
Nope. Copy paste.